src/cqrs/.template.config/template.json

```diff
@@ -12,7 +12,7 @@
 	"name": "Ensono Stacks CQRS Web API",
 	"identity": "Ensono.Stacks.CQRS.WebApi.CSharp",
 	"groupIdentity": "Ensono.Stacks.CQRS.WebApi",
-	"shortName": "stacks-cqrs-app",
+	"shortName": "stacks-cqrs",
```

Comment:
Does this rename it from the NuGet template? If so, WIN! We can ignore my previous concern.

Reply:
Yeah this renames the template to `stacks-cqrs`. I did it because the CLI stores `{{ .Project.Framework.Option }}` as `cqrs` so it was easier to update the NuGet template name instead of updating the CLI config.

Reply:
Perfect. Makes sense mate 😄